### ISSUE TITLE
Lower eyelid location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11553]
Preferred term label:
Lower eyelid location

Synonyms

    Inferior palpebral location

    Lower lid location

    Lower eyelid

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to the lower eyelid, the smaller, less mobile fold of skin and tissue positioned below the eye. In clinical genetics and ophthalmology, this modifier is frequently used to describe localized findings such as lower eyelid colobomas, lower eyelid ectropion, or epiblepharon. [1] PMID:30616908, PMID:37777491

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Treacher Collins syndrome (OMIM:154500) (e.g., lower eyelid coloboma/pseudocoloboma)

    Miller syndrome / Postaxial acrofacial dysostosis (OMIM:263750) (e.g., lower eyelid ectropion)

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
This is [Localized to the lower eyelid HP:6001265](https://hpo.jax.org/browse/term/HP:6001265)
Adding synonyms.
Thanks!
